guides/doc-Installing_Server_on_Red_Hat/master.adoc

```diff
@@ -1,6 +1,6 @@
 :imagesdir: common/images
 :numbered:
-ifeval::["{build}" != "satellite"]
+ifdef::foreman-el,foreman-deb,katello[]
```

Comment:
This is not working in my tests.

Reply:
Ahh, becuase the attributes file is loaded after this block, so it previously always worked by happy accident.